benchmark.py

```diff
@@ -29,7 +31,7 @@ def RunScenario(scenario_config):
 
   if scenario_config.browser is not None:
     browser_executable = utils.browsers_definition[scenario_config.browser]['executable']
-    if scenario_config.browser in ["Chrome", "Canary", "Edge"]:
+    if scenario_config.browser in ["Chromium", "Chrome", "Canary", "Edge"]:
```

Comment:
I don't think we want "open -a ..." for Chromium, since it's a full path, but rather "open ..."

Reply:
Humm, it seems to work fine as is. Do you know if it does anything bad to leave as is?

Reply:
Heh, if it works that's fine :)